README.rst

```diff
@@ -1,50 +1,62 @@
-# Atlassian JWT authentication
+============================
+Atlassian JWT authentication
+============================
 
-[![travis-status-image]][travis]
-[![pypi-version-image]][pypi]
+.. image:: https://img.shields.io/travis/atlassian/asap-authentication-python/master.svg?label=Linux%20build%20%40%20Travis%20CI
+   :target: http://travis-ci.org/atlassian/asap-authentication-python
+.. image:: https://img.shields.io/pypi/v/atlassian-jwt-auth.svg
+   :target: https://pypi.org/project/atlassian-jwt-auth
 
-This package provides an implementation of the [Service to Service Authentication](http://s2sauth.bitbucket.org/spec/) specification.
+This package provides an implementation of the `Service to Service Authentication <http://s2sauth.bitbucket.io/spec/>`_ specification.
```

Comment:
Pleasechange this to `https://s2sauth.bitbucket.io/spec/` .

Reply:
Sure, done!

P.S. JFYI: [It is possible to push in PR branch by yourself](https://github.com/blog/2247-improving-collaboration-with-forks), which is useful w/o need to wait until contributor does the addition of a single character by themself.